src/components/container/Container.js

```diff
@@ -0,0 +1,32 @@
+import { fetchData } from '../../apiCalls';
+import React, { useEffect, useState } from 'react';
+import Show from '../show_card/ShowCard'
+import './Container.css'
+
+const Container = (props) => {
+  const [ isLoading, setIsLoading ] = useState(true);
+  const [ shows, setShows ] = useState([]);
+
+  //TO DO: includes conditional so that we can use this for our tracks and our years. 
+  useEffect(() => {
+    if (props.type === "shows") {
+      fetchData(`/years/${props.id}`).then(data => {
+        setShows(data.data);
+        setIsLoading(false);
+      }).catch(err => console.log(err))
```

Comment:
We learned in the hooks lesson class yesterday the Resolving promises (.then) cannot be located within the useEffect method.  We will need to declare a function above the useEffect the handles the promise, then call that method in the useEffect